.github/workflows/Archive_Plugin_Artifact.yml

```diff
@@ -11,14 +11,14 @@ on:
 jobs:
   build:
     name: Build and archive
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
       - name: Checkout
         uses: actions/checkout@v3
       - uses: actions/setup-java@v3.3.0
         with:
           distribution: 'temurin'
-          java-version: '8'
+          java-version: '17'
```

Comment:
Seems a bit generous. We should probably target Java 11 since it is the minimal Java version Jenkins supports.